Dockerfile

```diff
@@ -1,63 +1,51 @@
-FROM node:16 as sourcebuild
-ARG ui_tag=main
-ARG api_tag=main
-
+# Frontend build
+FROM node:16 as FRONTEND
 WORKDIR /app
-RUN git clone --depth 1 --branch ${ui_tag} https://github.com/KaotoIO/kaoto-ui.git 
-RUN git clone --depth 1 --branch ${api_tag} https://github.com/KaotoIO/kaoto-backend.git 
-
+RUN git clone --depth 1 https://github.com/KaotoIO/kaoto-ui.git 
 WORKDIR /app/kaoto-ui
-RUN yarn install --mode=skip-build
-
+RUN yarn install
 RUN KAOTO_API="" yarn run build
 
-#--- backend build
-FROM registry.access.redhat.com/ubi8/openjdk-17 as kaotobuild
+#Backend build
+FROM quay.io/quarkus/ubi-quarkus-native-image:22.1-java17 as BACKEND
 USER root
-ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
-RUN mvn -version
-
-
-RUN mkdir /app \ 
-    &&  chown -R 1001 /app \
-     && chmod -R "g+rwX" /app \
-     && chown -R 1001:root /app 
-COPY --from=sourcebuild /app/kaoto-backend/  /app/kaoto-backend/
-WORKDIR /app/kaoto-backend
-
-COPY --from=sourcebuild /app/kaoto-ui/dist/ /app/kaoto-backend/api/src/main/resources/META-INF/resources/
-RUN mvn install -DskipTests=true -Dcheckstyle.skip=true
-
-
-# #--- kaoto run 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
-
-ARG JAVA_PACKAGE=java-17-openjdk-headless
-ARG RUN_JAVA_VERSION=1.3.8
-ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' 
-# Install java and the run-java script
-# Also set up permissions for user `1001`
-RUN microdnf install curl ca-certificates ${JAVA_PACKAGE} \
-    && microdnf update \
-    && microdnf clean all \
-    && mkdir /deployments \
-    && chown 1001 /deployments \
-    && chmod "g+rwX" /deployments \
-    && chown 1001:root /deployments \
-    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1001 /deployments/run-java.sh \
-    && chmod 540 /deployments/run-java.sh \
-    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/conf/security/java.security
-
-# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dquarkus.kubernetes-client.trust-certs=true"
```

Comment:
how are in native configured these properties ? it can be necessary for the running with k8

Reply:
```CMD ["./application", "-Dquarkus.http.host=0.0.0.0"]```